services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+import init_django_orm  # noqa: F401
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        cinema_hall_id: int,
+        movie_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(
+        session_date: str | None = None
+) -> QuerySet[MovieSession]:
+    if session_date:
+        queryset = MovieSession.objects.filter(show_time__date=session_date)
+        return queryset
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime.datetime | None = None,
+        movie_id: int | None = None,
+        cinema_hall_id: int | None = None
+) -> None:
+    movie_session = get_movie_session_by_id(movie_session_id=session_id)
+    if show_time:
+        movie_session.show_time = show_time
+    if movie_id:
+        movie_session.movie_id = movie_id
+    if cinema_hall_id:
+        movie_session.cinema_hall_id = cinema_hall_id
+    movie_session.save()
+
+
+def delete_movie_session_by_id(session_id: int) -> None:
+    movie_session = get_movie_session_by_id(movie_session_id=session_id)
+    movie_session.delete()
```

Comment:
There is no need to create `movie_session` variable
```suggestion
    get_movie_session_by_id(movie_session_id=session_id).delete()
```